pom.xml

```diff
@@ -240,6 +232,18 @@
                             <skipNoKey>false</skipNoKey>
                         </configuration>
                     </plugin>
+                    <plugin>
+                        <groupId>org.sonatype.central</groupId>
+                        <artifactId>central-publishing-maven-plugin</artifactId>
+                        <version>0.7.0</version>
+                        <extensions>true</extensions>
+                        <configuration>
+                            <publishingServerId>ossrh</publishingServerId>
+                            <autoPublish>true</autoPublish>
+                            <waitUntil>published</waitUntil>
+                            <deploymentName>Common custom user data maven extension</deploymentName>
```

Comment:
Any reason this value is not upper-cased as in `Common Custom User Data Maven Extension`? What is it used for?